engine/fake_exchange_test.go

```diff
@@ -96,13 +102,23 @@ func (h *FakePassingExchange) GetEnabledPairs(_ asset.Item) (currency.Pairs, err
 func (h *FakePassingExchange) GetAvailablePairs(_ asset.Item) (currency.Pairs, error) {
 	return currency.Pairs{}, nil
 }
+
 func (h *FakePassingExchange) FetchAccountInfo() (account.Holdings, error) {
-	return account.Holdings{}, nil
+	acc, err := account.GetHoldings(h.Name)
+	if err != nil {
+		return h.UpdateAccountInfo()
+	}
+
+	return acc, nil
 }
 
 func (h *FakePassingExchange) UpdateAccountInfo() (account.Holdings, error) {
-	return account.Holdings{}, nil
+	if err := account.Process(&h.Holdings); err != nil {
+		return account.Holdings{}, err
+	}
+	return h.Holdings, nil
```

Comment:
This can also just return 
```
return account.Holdings{
   Exchange: h.Name,
   Accounts: []account.SubAccount{},
}, nil
```

(or whatever is required for your rpcserver test)

Reply:
replaced